### PR TITLE
MNT: Nuke replay from dataportal. See NSLS-II/replay

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -28,8 +24,6 @@ requirements:
     - pandas
     - matplotlib
     - pyyaml
-    - enaml [py2k]
-    - xray-vision [py2k]
     - scikit-xray [py2k]
     - pims [py2k]
```
Comment (tacaswell) [inline conda-recipe/meta.yaml]: I think these deps are also here for replay's use

**Comment by danielballan** [comment]
And so it came to pass that, for the first time in weeks, Travis passed on dataportal. There was much rejoicing.

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (ericdill) [inline conda-recipe/meta.yaml]: skxray is used here `dataportal/examples/sample_data/frame_generators.py` and here `/dataportal/testing/decorators.py`

pims is used here `/dataportal/broker/pims_readers.py`

They should be left in the deps for sure

Comment (danielballan) [inline conda-recipe/meta.yaml]: Good catch, @ericdill. If the heavy deps are annoying (pims' conda package installs scipy!) maybe now is a good time to built a light pims-base package.

Comment (tacaswell) [inline conda-recipe/meta.yaml]: then they should probably lose the py2k filters

**Comment by ericdill** [comment]
k, good to merge?

**Pull request merged**